src/platform/nm-platform.c

```diff
@@ -57,22 +57,45 @@ G_STATIC_ASSERT (G_STRUCT_OFFSET (NMPlatformIPRoute, network_ptr) == G_STRUCT_OF
 
 #define _NMLOG_DOMAIN           LOGD_PLATFORM
 #define _NMLOG_PREFIX_NAME      "platform"
+
+
+#define NMLOG_COMMON(level, name, ...) \
+        char __prefix[32]; \
```

Comment:
let's always use a `G_STMT_START` block for multiline macros.

Reply:
Why? NMLOG_COMMON is *not* to be used mimicking a statement.